### DISCUSSION TITLE
Setting locale failed

### DISCUSSION THREAD

**Discussion by GeneralStarbound** [idaholab/moose#21118] [Q&A Getting Started] [answered]
When I try to run the compile and test code, which is as follows
cd ~/projects/moose/test
make -j 4
./run_tests -j 4
I keep getting an error that says that my setting locale failed, and that it will fall back on "c".  I am not sure what caused this bug or how to fix this.  Without this, I cannot tell if the program is running properly and it results in a mass of errors.

**Comment by cticenhour** [comment, marked as answer]
I have converted this issue to a discussion post, as this is more suited to that forum. I do not have enough information to give you suggestions and advice on what your issue is, so please let me know the following: 

- What is your platform? (Linux, Mac, Windows Subsystem for Linux - WSL)
- Are you using the MOOSE conda packages (following the [MOOSE Getting Started Instructions](https://mooseframework.inl.gov/getting_started/installation/index.html))?
- Is this an error during build, or only during testing? 
- Can you please post a copy of the error? You can either attach a plain `.txt` file to your reply, or copy and paste it in the post if it isn't too long. To get proper visual formatting, please use the code formatting syntax for GitHub noted [here](https://docs.github.com/en/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax#quoting-code).

**Reply by GeneralStarbound** [reply]
It wasn't important.  We got the program working.

**Reply by GeneralStarbound** [reply]
I do have a second question, probably also better for the discussion board, about using up too many slots

**Reply by cticenhour** [reply]
If you got it working, please post the solution in a new "answer" below so that it might help others. Feel free to post your other question as a new discussions post.
